dll/win32/comctl32/status.c

```diff
@@ -291,8 +291,18 @@ STATUSBAR_Refresh (STATUS_INFO *infoPtr, HDC hdc)
 
     SelectObject (hdc, hOldFont);
 
+#ifdef __REACTOS__
+    if (GetWindowLongW (infoPtr->Self, GWL_STYLE) & SBARS_SIZEGRIP) {
+        WINDOWPLACEMENT wndPlacement;
+        wndPlacement.length = sizeof(WINDOWPLACEMENT);
+
+        if (!GetWindowPlacement (infoPtr->Notify, &wndPlacement) || wndPlacement.showCmd != SW_MAXIMIZE)
```

Comment:
Is `infoPtr->Notify` always the correct window or would `GetParent` be more appropriate?

Reply:
You can use WS_MAXIMIZE window style.

Reply:
@ThFabba Yes. As specified in `CREATESTRUCTA` struct.
@katahiromz I will try that.